src/views/ProjectView.vue

```diff
@@ -26,85 +27,99 @@ const noSelectedImages = computed(() => {
   return store.getters.selectedImages.length === 0
 })
 
+// manages successful api response by mapping data to unique sessions
+const handleSuccess = (data) => {
```

Comment:
Now that I think about this more, maybe we can rename this function to what it's doing since we're only using it for a single API call? handleSuccess isn't immediately descriptive